### PR TITLE
sstimer: Does a thing in a less retarded way.

### PR DESCRIPTION
**Pull request by MrsTonedOne**

### CODE REVIEW

File: code/controllers/subsystem/timer.dm [hunk 1]
```diff
@@ -83,9 +83,8 @@ var/datum/subsystem/timer/SStimer
 			return hash_event.id
 	SStimer.hashes[event.hash] = event
 	if (wait <= 0)
-		spawn
```
Comment (Jordie0608) [inline code/controllers/subsystem/timer.dm]: `spawn`
Huh? Is this just spawn(0)?

Comment (MrStonedOne) [inline code/controllers/subsystem/timer.dm]: yes

**Comment by optimumtact** [comment]
:grin:

**Comment by MrStonedOne** [comment]
so basically, after this pr, addtimer(thing,proc,0) is the same as calling it with set waitfor = 0 (with a tiny bit more overhead)

**Pull request merged**